evalml/demos/breast_cancer.py

```diff
@@ -1,8 +1,9 @@
 import pandas as pd
+import woodwork as ww
 from sklearn.datasets import load_breast_cancer as load_breast_cancer_sk
 
 
-def load_breast_cancer():
+def load_breast_cancer(return_pandas=False):
```

Comment:
I decided to go with adding a parameter because it'd still be nice to give users that flexibility, but lmk any thoughts!

Reply:
@dsherry Thanks! I looked at the link and am going to update this to 
```
Union[(ww.DataTable, ww.DataColumn), (pd.Dataframe, pd.Series)]: X and y
```

I think this makes a bit more sense because it's more clear that it either returns two Woodwork or two pandas, but lmk if you think otherwise!

Reply:
@angela97lin yep that looks good! Our sphinx API ref gen doesn't do any sort of parsing of return type currently, so I'm on board with whatever is communicative and more or less in-line with the spec :)